docs/en/stack/ml/nlp/ml-nlp-e5.asciidoc

```diff
@@ -0,0 +1,161 @@
+[[ml-nlp-e5]]
+= E5 – EmbEddings from bidirEctional Encoder rEpresentations
+++++
+<titleabbrev>E5</titleabbrev>
+++++
+
+:frontmatter-description: E5 is a multilingual dense vector model.
+:frontmatter-tags-products: [ml] 
+:frontmatter-tags-content-type: [how-to] 
+:frontmatter-tags-user-goals: [analyze]
+
+experimental[]
```

Comment:
Experimental or not is an open question, please wait for confirmation before merging